hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/INodeAttributeProvider.java

```diff
@@ -405,6 +405,30 @@ default void checkPermissionWithContext(AuthorizationContext authzContext)
           + "implement the checkPermissionWithContext(AuthorizationContext) "
           + "API.");
     }
+
```

Comment:
FSPermissionChecker#isSuperUser() is called in ContentSummaryComputationContext#checkPermission(), should we add a FSPermissionChecker#checkSuperuserPrivilege for auditing?